cli/command/cli.go

```diff
@@ -151,6 +158,89 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 	return nil
 }
 
+// NotaryClient provides a Notary Repository to interact with signed metadata for an image
+func (cli *DockerCli) NotaryClient(imgRefAndAuth ImageRefAndAuth, forWrite bool) (notaryclient.Repository, error) {
```

Comment:
I always try to avoid boolean arguments because from the caller it's not really clear what `true` or `false` means. Maybe this could takes a `[]string` of actions, and there could be two predefined slices in `cli/trust` 

```go

var (
  ActionsPullOnly = []string{"pull"}
  ActionsPushAndPull = []string{"pull", "push"}
)
```
So from the caller it will be obvious what is being requested.